auslib/web/admin/views/scheduled_changes.py

```diff
@@ -50,15 +50,16 @@ def get(self):
                 # We don't need to consider the existing version of a row for
                 # inserts, because it doesn't exist yet!
                 if row["change_type"] != "insert":
-                    affected_rows.append(self.table.select(where=base_pk)[0])
+                    base_row = self.table.select(where=base_pk)[0]
+                    scheduled_change["base_row"] = base_row
```

Comment:
It looks like this is changing the schema of the responses from from /scheduled_changes/rules et. al. You should update https://github.com/mozilla/balrog/blob/master/auslib/web/admin/swagger/api.yaml#L1815 to reflect this.

Reply:
Yep, thanks!